webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const HtmlWebPackPlugin = require('html-webpack-plugin');
+const path = require('path');
 
```

Comment:
path was imported but not used anywhere.

Reply:
Oh thanks for the insight about that, i forgot it during my first configuration.